vng_api_common/pagination.py

```diff
@@ -13,6 +13,9 @@ class DynamicPageSizeMixin:
     page_size = 100
     page_size_query_param = "pageSize"
     max_page_size = 500
+    page_size_query_description = _(
+        "Het aantal resultaten terug te geven per pagina. (default: 100)."
+    )
```

Comment:
Would it be possible to have the page_size be based on the value of the `page_size` attribute? Since classes that subclass this can change the default.

Perhaps something like this could work (not 100% sure though):

```suggestion
        @property
        def page_size_query_description(self):
             return _("Het aantal resultaten terug te geven per pagina. (default: {page_size}).").format(page_size=self.page_size)
    ))
```